src/iam.ts

```diff
@@ -549,6 +557,34 @@ export class IAM extends IAMBase {
         throw new Error(ERROR_MESSAGES.PROVIDER_NOT_INITIALIZED);
     }
 
+    /**
+     * createIdentityProofWithDelegate
+     *
+     * @description create a raw identity proof for a delegate
+     * @returns JWT token of created identity
+     *
+     */
+    async createIdentityProofWithDelegate(
+        secp256k1PrivateKey: string,
+        rpcUrl: string,
+        identityProofDid: string,
+    ): Promise<string> {
+        const provider = new providers.JsonRpcProvider(rpcUrl);
+        const wallet = new Wallet(secp256k1PrivateKey, provider);
```

Comment:
Wallet is not really needed to sign. Is it possible to sign just with `jsonwebtoken` npm package?